room-battle.js

```diff
@@ -346,6 +346,8 @@ class Battle {
 		this.id = room.id;
 		/** @type {GameRoom} */
 		this.room = room;
+		/** @type {Boolean} */
+		this.reported = false;
```

Comment:
Hm, can you do this without the extra variable?

Reply:
Not sure how I would prevent battles from being reported twice without something to check.
If there already is something to check that you would know of, let me know.

Reply:
I believe `this.started` may be sufficient.

Reply:
I assumed that would be set to true in an imported replay, but it seems your right, that will work for this.